### ISSUE TITLE
"Validate that get-help <cmdletName> works"  test requires sudo to run on linux

### ISSUE THREAD

**Issue by TravisEz13** [PowerShell/PowerShell#3527]
Steps to reproduce
------------------
Start-PSPester -binDir /opt/microsoft/powershell/6.0.0-alpha.18

Expected behavior
-----------------
Tests should work

Actual behavior
---------------
Test at [HelpSystem.Tests.ps1#L76](https://github.com/PowerShell/PowerShell/blob/da2fd5c89fcbd9f8837988b4cc708b08b8211666/test/powershell/engine/Help/HelpSystem.Tests.ps1#L76)
fails with:
```
Failed to update Help for the module(s) 'Microsoft.PowerShell.Core` with UI culture(s) {en-US} : Access to the path '/opt/microsoft/powershell/6.0.0-alpha.18/en-US/System.Management.Automation.dll-help.xml' is denied.
```
![image](https://cloud.githubusercontent.com/assets/10873629/24886197/bde6b4c0-1e07-11e7-90af-f7e681cd595a.png)

## Additional tests failing similarly

- [ HelpSystem.Tests.ps1](https://github.com/PowerShell/PowerShell/blob/da2fd5c89fcbd9f8837988b4cc708b08b8211666/test/powershell/engine/Help/HelpSystem.Tests.ps1)
    - Validate that Get-Help returns provider-specific help
    - Validate about_help.txt under culture specific folder works
- [UpdatableHelpSystem.Tests.ps1](https://github.com/PowerShell/PowerShell/blob/9aa693dfb31c0c310db9504dc1dbc4b0d3b62fd3/test/powershell/engine/Help/UpdatableHelpSystem.Tests.ps1)
    - Validate Update-Help -SourcePath for one PowerShell Core module.

Environment data
----------------

```powershell
> $PSVersionTable
Name                           Value
----                           -----
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0...}
GitCommitId                    v6.0.0-alpha.18
PSVersion                      6.0.0-alpha
PSRemotingProtocolVersion      2.3
WSManStackVersion              3.0
SerializationVersion           1.1.0.1
PSEdition                      Core
BuildVersion                   3.0.0.0
CLRVersion
```

**Comment by joeyaiello** [comment]
Probably blocked by #2190

**Comment by TravisEz13** [comment]
@joeyaiello Another solution would be to tag these test as requiring `sudo` until the issue is resolved.

**Comment by joeyaiello** [comment]
Didn't realize we had a mechanism for that. I'm all for it. :)

**Comment by TravisEz13** [comment]
Since I'm running the tests manually... a new tag would work.

**Comment by kalgiz** [comment]
Fixed in #6223
